src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -71,22 +88,48 @@ public void indexSnapshotTest() throws Exception {
         assertEquals(indexName, searchIndexModel.getIndexSummary());
     }
 
+    @Test
+    public void querySnapshotTest() throws Exception {
+        GetAliasesResponse mockResponse = mock(GetAliasesResponse.class);
+        when(mockResponse.getAliases()).thenReturn(Map.of("0f14d0ab-9605-4a62-a9e4-5ed26688389b", Set.of()));
```

Comment:
Is this the name of the index alias? If so, it should be `idx-<id>` (see `indexName`), correct?

Reply:
it doesn't actually affect the test, but I can change it to be more accurate to indexing behavior.